src/functions-and-arrays.js

```diff
@@ -1,73 +1,431 @@
 // Iteration #1: Find the maximum
 
+var maxOfTwoNumbers = [5, 7];
+
+function findMax(array) {
```

Comment:
- The function should have two separate arguments, not an array as an argument.
- The function is named incorrectly and is not logging the result if the second number is highest.